gradle/javaProject.gradle

```diff
@@ -0,0 +1,83 @@
+apply plugin: "java"
+apply plugin: "com.github.spotbugs"
+apply plugin: "checkstyle"
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+
+dependencies {
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.8.6'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.8.6'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.8.0'
+    implementation group: 'org.ballerinalang', name: 'ballerina-lang', version: '1.2.4'
+    implementation group: 'org.ballerinalang', name: 'ballerina-test-utils', version: '1.2.4'
+    testCompile 'org.testng:testng:6.13.1'
+
+    checkstyle project(":build-config:checkstyle")
+    checkstyle "com.puppycrawl.tools:checkstyle:${puppycrawlCheckstyleVersion}"
+}
+
+test {
+    systemProperty "java.util.logging.config.file", "src/test/resources/logging.properties"
+    systemProperty "java.util.logging.manager", "org.ballerinalang.logging.BLogManager"
+    systemProperty "ballerina.home", "/home/miyurud/software/ballerina-1.2.4/distributions/jballerina-1.2.4"
```

Comment:
Will this break the tests ?

Reply:
@miyurud will be working on this, just copied test-task from his build gradle.

Reply:
Yes, I will fix this issue